js/package.json

```diff
@@ -24,11 +24,10 @@
     "README.md"
   ],
   "scripts": {
-    "build": "rm -rf dist && npx tsc --project tsconfig.cjs.json && npx tsc --project tsconfig.esm.json",
-    "fmt": "npx prettier --write . && npx eslint --fix .",
-    "lint": "npx prettier --check . && npx eslint .",
-    "test": "DL_SKIP_SSL_VERIFICATION=1 NODE_EXTRA_CA_CERTS=\"$(mkcert -CAROOT)/rootCA.pem\" npx jest",
-    "typecheck": "npx tsc --project tsconfig.eslint.json && npx tsc --project spec/tsconfig.json"
+    "build": "rm -rf dist && tsc --project tsconfig.cjs.json && tsc --project tsconfig.esm.json",
+    "fmt": "prettier --write . && eslint --fix .",
+    "lint": "prettier --check . && eslint . && tsc --project tsconfig.eslint.json && tsc --project spec/tsconfig.json",
+    "test": "DL_SKIP_SSL_VERIFICATION=1 NODE_EXTRA_CA_CERTS=\"$(mkcert -CAROOT)/rootCA.pem\" jest"
```

Comment:
Removed the `npx` prefixes because they aren't needed. Also merged `lint` and `typecheck` because they're basically the same thing.